dev/sample/.vscode/launch.json

```diff
@@ -0,0 +1,58 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "compounds": [
+    {
+      "name": "Electron Webpack: Launch Main & Attach Renderer",
+      "configurations": [
+        "Electron Webpack: Launch Main",
+        "Electron Webpack: Attach Renderer"
+      ]
+    }
+  ],
+  "configurations": [
+    {
+      "name": "Electron Webpack: Attach Renderer",
+      "type": "chrome",
+      "request": "attach",
+      "port": 9223,
+      "webRoot": "${workspaceFolder}",
+      "sourceMaps": true,
+      "showAsyncStacks": true,
+      "timeout": 30000
+    },
+    // Configuration adapted from https://github.com/electron-userland/electron-webpack/issues/80.
+    {
+      "name": "Electron Webpack: Launch Main",
+      "type": "node",
+      "request": "launch",
+      "cwd": "${workspaceFolder}",
+      "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/electron-webpack",
+      "args": ["dev"],
+      "env": {
+        //"DEBUG":"*",
+        "ELECTRON_DISABLE_SECURITY_WARNINGS": "1"
+      },
+      "autoAttachChildProcesses": true,
+      "console": "integratedTerminal",
+      "outputCapture": "std",
+      "sourceMaps": true,
+      "smartStep": true,
+      "showAsyncStacks": true,
+      "timeout": 30000
```

Comment:
This timeout will vary, but it needs to be _longer_ than the amount of time it takes for the electron app to launch so that VSCode does not give up. Perhaps i will note this..

Reply:
noted.